src/components/Timeline/index.js

```diff
@@ -0,0 +1 @@
+export {default} from "./Timeline"
```

Comment:
Be careful on empty line at the end of files!

Reply:
palmface

Reply:
Also, I should really setup some sort of linter in the CI